lib/errors/api_error_handling.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ApiErrorHandling
+  def log_error(error, update_service: nil, sentry_extra:)
+    Rails.logger.info "Caught #{error}"
+    sentry_tags = nil
+    if update_service.present?
+      update_service.update_error_stats
```

Comment:
This behavior isn't being tested at all (and shows up as uncovered code after a spec run).

You should add appropriate tests, perhaps to `update_course_stats_spec.rb`, to verify that it does what we expect in terms of posting something to Sentry and including the right error count in the `flags` update logs.

`constant_update_spec.rb` also has some relevant examples, in terms of interacting with Sentry behavior in a test.

Reply:
Done!